### DISCUSSION TITLE
Dashboard stuck connecting to database

### DISCUSSION THREAD

**Discussion by sarvasvkulpati** [supabase/supabase#6637] [Questions]
For some reason, my database is stuck connecting. It keeps saying 'This may take a few minutes, but if your dashboard hasn't connected within 2 minutes, you can open a support ticket'

My entire website is down after just launching. How can I fix this, it's urgent!

<img width="619" alt="image" src="https://user-images.githubusercontent.com/23270560/166133305-1db98f23-48e0-4367-9ac2-2018f30ae619.png">

**Comment by kav** [comment]
Seeing the same on one of our projects as well but not the other. If you check the console there is a CORS error:
Access to fetch at 'https://<project_id>.supabase.co/rest/v1/' from origin 'https://app.supabase.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Reply by ru88s** [reply]
Same problem.
I have restarted the server but it does not fix it.

**Reply by bhagavan44** [reply]
Same for me too, restarted the server but issue remains same

**Reply by kav** [reply]
Sounds like there is currently an issue with AWS DNS https://status.supabase.com/

**Reply by GaryAustin1** [reply]
Yes the issue you are having tonight has nothing to do with the issue of this discussion and was as linked by @kav.

**Reply by bhagavan44** [reply]
Actually this was not started today, it is like this from past 2 days. This server was paused some time back, I have resumed it 2 days back since then it showing this connecting screen only.

**Reply by troyshu** [reply]
Restarting the server didn't work for me either: I'm still stuck on "Connecting" on many of the supabase dashboard pages, like "Database", "SQL Editor", etc. My server was also paused a little bit ago, and now unpaused. Am contacting support and will report back what I discover.

**Comment by kav** [comment]
Resolved the issue with quick help from support by turning it off and then on again. I.e. go to settings in the dashboard and restart the server

**Reply by longnguyen11-tiki** [reply]
Hello @kav , how to set request mode to 'no-cors' ?
Thank you for your support!

**Reply by kav** [reply]
I didn't. I just restarted the server using the button in settings and that resolved the issue

**Comment by NorkzYT** [comment]
> For some reason, my database is stuck connecting. It keeps saying 'This may take a few minutes, but if your dashboard hasn't connected within 2 minutes, you can open a support ticket'
> 
> My entire website is down after just launching. How can I fix this, it's urgent!
> 
> <img alt="image" width="619" src="https://user-images.githubusercontent.com/23270560/166133305-1db98f23-48e0-4367-9ac2-2018f30ae619.png">

I am having the exact same issues. I did not find the settings page that @kav mentioned.

**Reply by kav** [reply]
It's under Settings, General. I've found it tends to happen if the server runs out of memory, so you might consider upping the instance size and memory in particular

**Reply by NorkzYT** [reply]
Could you send me an image, I can't seem to find it.

**Reply by troyshu** [reply]
To restart the server: Gear Icon → General (under Project Settings) → Restart server (under Infrastructure)

![CleanShot 2022-11-04 at 10 13 29](https://user-images.githubusercontent.com/1057286/199996433-15b3b772-eecf-4d3c-b060-4f85b91fc981.png)

**Reply by NorkzYT** [reply]
Understood, sadly I am not able to access the Gear Icon since dashboard is stuck on `Connecting to Default Project`.

**Comment by troyshu** [comment]
@sarvasvkulpati or @bhagavan44 did you ever resolve your issues? I'm facing a similar issue (if not the same) right now too. In fact, a lot of the other dashboard pages are stuck on "Connecting", including "Table Editor", "SQL Editor", "Database", "Authentication". The page to reset the JWT secret is stuck on "Connecting" too (and my JWT secret expired so I need to regenerate them to be able to use the API but can't, at least through the dashboard). Restarting the server/project in Settings doesn't resolve the error.

I am able to use psql to connect to my project's database and see that all the data is still there.

I also see some 401 errors in the console logs from what it looks like the dashboard trying to auth with the live rest API for my project:

![CleanShot 2022-11-04 at 11 55 32](https://user-images.githubusercontent.com/1057286/200020281-ba68026d-3309-473b-90b7-63e09b902757.png)

Anyone have any ideas of what might be going on here? Because the dashboard is stuck on "Connecting", I'm unable to re-generate my JWT secret (which somehow expired) so the supabase calls I'm making in my live product aren't working right now.

**Reply by NorkzYT** [reply]
Checkout the following issue.
https://github.com/supabase/supabase/issues/9775

**Reply by troyshu** [reply]
Thanks. Not sure how that issue is related though? The issues described here are for the supabase hosted dashboard (log in at https://supabase.com/), not self-hosted

**Reply by NorkzYT** [reply]
The issues original post is called, `API calls that fail (404 error) in self-hosted supabase studio` which does deal with self-hosting.
https://github.com/supabase/supabase/issues/8721#issuecomment-1287867693

**Reply by bhagavan44** [reply]
@troyshu No, issue not resolved for me. Tried the server restart several times with no luck. I have created the new instance to move forward and deleted the old one.

**Reply by troyshu** [reply]
Supabase support just resolved the issue for me, was some sort of technical issue on their side. All my data, users, functions, etc. were still there. I got on their Pro tier (right after I had encountered the issue). Good luck!

**Comment by akshg05** [comment]
Hello, I'm facing a similar issue. My project was paused and when resumed back, it's stuck to this connecting screen and console is filled with similar CORS errors to <projectid>,supabase.co/../../live.
Can someone please help me with a speedy resolution. I am in need the data of the project and can't wait for days. I'm on the free plan.

**Reply by GaryAustin1** [reply]
Please contact support if you have not.

**Reply by akshg05** [reply]
Yeah I did yesterday. No response yet. If only I knew pausing and resuming could lead to this, I'd not let it pause the first place.

**Comment by samuyoon** [comment]
Experiencing this same issue in March 2023. I've emailed support, but I'm on the free tier so not sure how quicky I'm going to see a resolution. Does anyone have any ideas?

**Comment by noty-yura** [comment]
Got the same issue

**Comment by Mushfiq2002** [comment]
Faced the same issue 
contacted support 
still no reply
